### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by GuobinZhangTJU** [MIC-DKFZ/nnUNet#344]
Hi, Fabian, Can you give me some advice? I meet a question when used your nnunet to train a multi-organ segmentation model.

The dataset I used have a characteristic: in each case, the labeled organ 1 occured in many slices but labeld organ 2 only occured in few slices because the organ 2 has a small volumel. Specific performance is that false positives are making more and the dice of organ 2 is not very well.

The nnunet automatic adopt dice+CE as the loss function, can you give me some advices about loss function to solve this problem~

**Comment by FabianIsensee** [comment]
Hi,
I cannot give any advice without knowing the dataset. It is unlikely that the loss is at fault: we had pretty good performance on the Beyong the Cranial Vault challenge where the task was also multiorgan segmentation (some large, some small organs).
Best,
Fabian

**Comment by GuobinZhangTJU** [comment]
> Hi,
> I cannot give any advice without knowing the dataset. It is unlikely that the loss is at fault: we had pretty good performance on the Abdominal Organ Segmentations Beyond the Cranial Vault challenge where the task was also multiorgan segmentation (some large, some small organs).
> Best,
> Fabian

got it ~ many thanks.
